### ISSUE TITLE
LibraryAppletController not implemented

### ISSUE THREAD

**Issue by HorrorTroll** [Ryujinx/Ryujinx#648]
LibraryAppletController is not implemented

## Applet description:
```
This is the applet presented when launching a game that allows the user to select a controller type
```

## Attention:
```
This applet doesn't have document yet!
```
 
## Required by:
* https://github.com/Ryujinx/Ryujinx-Games-List/issues/153
* https://github.com/Ryujinx/Ryujinx-Games-List/issues/181
* https://github.com/Ryujinx/Ryujinx-Games-List/issues/229
* https://github.com/Ryujinx/Ryujinx-Games-List/issues/227
* https://github.com/Ryujinx/Ryujinx-Games-List/issues/252

**Comment by ironlink17** [comment]
This may also be requested by Astral Chain, as you can see here:
[Ryujinx Log [Astral Chain]](https://github.com/Ryujinx/Ryujinx/files/4332628/Ryujinx_2020-03-14_12-15-00.log)

**Comment by B3astTech** [comment]
https://github.com/Ryujinx/Ryujinx-Games-List/issues/533

**Comment by B3astTech** [comment]
https://github.com/Ryujinx/Ryujinx-Games-List/issues/538

**Comment by B3astTech** [comment]
https://github.com/Ryujinx/Ryujinx-Games-List/issues/539

**Comment by AcK77** [comment]
Fixed in #1003
